README.md

```diff
@@ -17,55 +20,63 @@ CRB service provides capabilities such as:
 The functionality above is described using OpenAPI specification by [swagger.yaml](swagger.yaml) through [Swagger framework](https://swagger.io).
 
 # Functional Tests
-[functionaltests](/functionaltests) contains the functional tests that validate the functionality of the CRB using [REST API](swagger.yaml)
+[functionaltests](/functionaltests) contains the functional tests that validate the functionality of the CRB using [REST API](swagger.yaml).
 
 # How to build
-The crb binary is built using a [container](Dockerfile) which has the required build tools.
+The CRB binary is built using a Docker [container](Dockerfile) which has the required build tools.
 
-* git clone https://github.com/ocopea/pcf-fs-crb.git crb.
+* git clone https://github.com/ocopea/pcf-fs-crb.git CRB.

```

Comment:
folder name is lower case crb. Please revert.

Reply:
done